modules/text2vec-openai/clients/openai.go

```diff
@@ -243,6 +244,47 @@ func (v *client) getOpenAIOrganization(ctx context.Context) string {
 	return v.openAIOrganization
 }
 
+func (v *client) getRateLimit(ctx context.Context) (int, int) {
+	returnRPM := 0
+	returnTPM := 0
+	if rpmS := v.getValueFromContext(ctx, "X-Openai-Ratelimit-RequestPM-Embedding"); rpmS != "" {
```

Comment:
In order for this to work header name needs to be of this format: `X-Openai-Ratelimit-Requestpm-Embedding`

Reply:
see above